### PR TITLE
[Lottie exporter] Implement use-defs

### PR DESCRIPTION
**Pull request by AnishGG**
Fixes: #1243
Fixes: #1242

I have tested some sample files, and this fix works for them. @rodolforg can you please review this. ^__^

### CODE REVIEW

**Comment by AnishGG** [comment]
So the lottie-build fails for this. Let me check it out.

**Comment by AnishGG** [comment]
> So the lottie-build fails for this. Let me check it out.

It is fixed now.

File: synfig-studio/plugins/lottie-exporter/common/Param.py [hunk 1]
```diff
@@ -44,10 +44,34 @@ def get_exported_valuenode(self):
...
             return
+
+        # Full ID string format is like: [[file-path]#][canvas-id:][child-canvas-id:][child-canvas-id:]…value-node-id
+        # Refer here: https://github.com/synfig/synfig/issues/993
         layer = self.get_layer()
         canvas = layer.getparent()
         key = self.param.attrib["use"]
-        anim = canvas.get_def(key)
+        if ":" in key:  # Start searching from root canvas
+            canvas = settings.ROOT_CANVAS
```
Comment (rodolforg) [inline synfig-studio/plugins/lottie-exporter/common/Param.py:54]: Wouldn't it be safer to check (later) whether first token keys[0 or 1].split(":") is empty?

There shouldn't be "something::other" in id, I guess.

Comment (AnishGG) [reply synfig-studio/plugins/lottie-exporter/common/Param.py:54]: I am not sure what you are trying to say here. Do you mean that the file name can have a `:` inside it? If yes, then I will need to put this check later.
Is this what you mean here?

Comment (AnishGG) [reply synfig-studio/plugins/lottie-exporter/common/Param.py]: Made the necessary changes.

**Comment by rodolforg** [comment]
That error seems to be gone, but I got another one while testing with that `walk.sif` example file:
Captura de tela de 2021-06-11 06-24-59


Are they related?

**Comment by AnishGG** [comment]
> Are they related?

Nope, this a different but known error.

`079-ATF-skeleton-animated-spline.sif` test file fails with the same error. You can have a look at the same here: https://travis-ci.com/github/AnishGG/synfig/jobs/512991945#L1677

**Review by rodolforg — APPROVED** [review]

**Comment by AnishGG** [comment]
@ice0 I think this PR is ready for merging. 😀

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**